packages/api/src/utils/magic.numbers.ts

```diff
@@ -0,0 +1,14 @@
+export enum CompanyStatus {
+  Active = "active",
+  Deleted = "deleted",
+}
+
+export enum UserCompanyRole {
+  Owner = 0,
+  Admin = 10,
+  Editor = 20,
+  Contributor = 30,
+}
+
+export const sessionTokenExpireDays = 7;
+export const sessionTokenStringLength = 128;
```

Comment:
Usually, the convention is to use all caps for hardcoded fixed vars.

```suggestion
export const SESSION_TOKEN_EXPIRE_DAYS = 7;
export const SESSION_TOKEN_STRING_LENGTH = 128;
```